controllers/syncer/csi_node.go

```diff
@@ -342,42 +339,30 @@ func (s *csiNodeSyncer) ensureVolumes() []corev1.Volume {
 	}
 }
 
-func (s *csiNodeSyncer) getSidecarByName(name string) *csiv1.CSISidecar {
-	return getSidecarByName(s.driver, name)
-}
-
-func (s *csiNodeSyncer) getCSINodeDriverRegistrarImage() string {
-	sidecar := s.getSidecarByName(config.CSINodeDriverRegistrar)
-	if sidecar != nil {
-		return fmt.Sprintf("%s:%s", sidecar.Repository, sidecar.Tag)
-	}
-	return "us.gcr.io/k8s-artifacts-prod/sig-storage/csi-node-driver-registrar:v2.3.0"
-	//return s.driver.GetDefaultSidecarImageByName(config.CSINodeDriverRegistrar)
-}
-
-func (s *csiNodeSyncer) getLivenessProbeImage() string {
-	sidecar := s.getSidecarByName(config.LivenessProbe)
-	if sidecar != nil {
-		return fmt.Sprintf("%s:%s", sidecar.Repository, sidecar.Tag)
-	}
-	return "us.gcr.io/k8s-artifacts-prod/sig-storage/livenessprobe:v2.4.0"
-	//return s.driver.GetDefaultSidecarImageByName(config.LivenessProbe)
-}
-
-func (s *csiNodeSyncer) getCSINodeDriverRegistrarPullPolicy() corev1.PullPolicy {
-	sidecar := s.getSidecarByName(config.CSINodeDriverRegistrar)
-	if sidecar != nil && sidecar.ImagePullPolicy != "" {
-		return sidecar.ImagePullPolicy
-	}
-	return corev1.PullIfNotPresent
-}
+func (s *csiNodeSyncer) getImage(name string) string {
```

Comment:
let's add a doc string, stating the purpose of the method and return type.

Reply:
I had added the description before the control logic, removed and added improved doc/description before method definition, which I agree is a better place for this. Thanks for pointing this out.